public/menu.js

```diff
@@ -53,7 +53,7 @@ module.exports = ({ app, mainWindow, newVersion, isStoreBuild }) => {
           label: esc(t('Import project')),
           submenu: [
             {
-              label: esc(t('Times in seconds (CSV)')),
+              label: esc(t('Times in seconds / Timestamps (CSV)')),
```

Comment:
If this breaks all translations, I think it's okay not to change the label of this menu item.